be/src/pipeline/pipeline_fragment_context.cpp

```diff
@@ -1568,7 +1586,7 @@ Status PipelineFragmentContext::send_report(bool done) {
     ReportStatusRequest req {true,
                              exec_status,
                              runtime_states,
-                             nullptr,
+                             _runtime_profile.get(),
```

Comment:
why it is nullptr in the past?

Reply:
This is for non-pipeline profile if pipeline mode is off

Reply:
will non-pipeline go into this method?

Reply:
> will non-pipeline go into this method?

No. But non-pipeline will use this `ReportStatusRequest ` struct so we keep this field here.